### PR TITLE
CODEOWNERS: add hubble as codeowner for hubble related helm charts

### PR DESCRIPTION
**Pull request by rolinh**

### CODE REVIEW

**Review by gandro — APPROVED** [review]

**Review by pchaigno — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -77,6 +77,8 @@ Dockerfile* @cilium/build
 /hubble-relay.Dockerfile @cilium/hubble
 /images @cilium/build
 /install/kubernetes/ @cilium/kubernetes @cilium/docs @cilium/helm
+/install/kubernetes/cilium/charts/hubble-relay @cilium/hubble
+/install/kubernetes/cilium/charts/hubble-ui @cilium/hubble
```
Comment (pchaigno) [inline CODEOWNERS]: Should @cilium/helm also be a codeowner on these files?

Comment (rolinh) [reply CODEOWNERS]: @cilium/helm is already a codeowner of `/install/kubernetes/`, see line above.

Comment (rolinh) [reply CODEOWNERS]: Oh wait, are you saying that the later match takes precedence so this group would be excluded? If yes then I'll add @cilium/helm

Comment (pchaigno) [reply CODEOWNERS]: Yes, the later match takes precedence and overwrites any previous match.

Comment (pchaigno) [reply CODEOWNERS]: Maybe we also need to add @cilium/kubernetes, though it's starting to be quite a party...

Comment (rolinh) [reply CODEOWNERS]: Updated so that it matches the groups from parent + @cilium/hubble.

**Pull request merged**